dessia_common/workflow/blocks.py

```diff
@@ -67,18 +63,20 @@ def output_from_function(function, name: str = "result output"):
 
 
 class BlockError(Exception):
-    pass
+    """ Specific BlockError Exception. """
```

Comment:
pass is not required ?

Reply:
Well, I was a bit confused as well, but I got yelled at by pylint because of "unecessary pass". So I guess it is not if you set a docstring, which is pretty cool in my opinion

Reply:
Yes that's nice !
Pylint is a nice teacher !